feil-behandler/src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/feilbehandler/river/FeilLytter.kt

```diff
@@ -58,7 +63,7 @@ class FeilLytter(
             sikkerLogger.warn("Kunne ikke parse feil-objekt, ignorerer...")
             return
         }
-        if (skalHaandteres(fail)) {
+        if (behovSkalHaandteres(fail) || eventSkalHaandteres(fail)) {
```

Comment:
Har gått for en enklest mulig måte å koble på feillytteren på Berik-tjenesten, ved å plukke opp eventet `INNTEKTSMELDING_SKJEMA_LAGRET`, som er det eventet som Beriktjenesten bruker gjennom stegene sine (og som ikke brukes av noen andre tjenester).